test/test_service.rb

```diff
@@ -109,4 +109,28 @@ def test_get_service
                      'http://localhost:8080/api/v1beta3/namespaces/development/services/redis-slave',
                      times: 1)
   end
+
+  def test_update_service
```

Comment:
I won't continue here, sorry but I think your idea of testing is not how it's meant to be. The test cases will never run against a real k8s server. Just merge this, it's better than the current behavior with the wrong name variable and has absolute no testing of updates at all.

Hope you can understand my point here....

Cheers